assets/queries/cloudFormation/docdb_cluster_master_user_password_rule/test/positive.yaml

```diff
@@ -0,0 +1,88 @@
+Resources:
+  NewAmpApp:
+    Type:  AWS::DocDB::DBCluster
+    Properties:
+      AvailabilityZones:
+        - String
+      BackupRetentionPeriod: Integer
+      DBClusterIdentifier: String
+      DBClusterParameterGroupName: String
+      DBSubnetGroupName: String
+      DeletionProtection: true
+      EnableCloudwatchLogsExports:
+        - String
+      EngineVersion: String
+      KmsKeyId: String
+      MasterUsername: admin
+      MasterUserPassword: 'asDjskjs73!!'
+      Port: Integer
+      PreferredBackupWindow: String
+      PreferredMaintenanceWindow: String
+      SnapshotIdentifier: String
+      StorageEncrypted: true
+      VpcSecurityGroupIds:
+        - String
+---
+Parameters:
+  ParentMasterPassword:
+    Description: 'Password'
+    Type: String
+    Default: 'asDjskjs73!'
+  ParentMasterUsername:
+    Description: 'username'
+    Type: String
+    Default: 'username!'
+Resources:
+  NewAmpApp-1:
+    Type: AWS::DocDB::DBCluster
+    Properties:
+      AvailabilityZones:
+        - String
+      BackupRetentionPeriod: Integer
+      DBClusterIdentifier: String
+      DBClusterParameterGroupName: String
+      DBSubnetGroupName: String
+      DeletionProtection: Boolean
+      EnableCloudwatchLogsExports:
+        - String
+      EngineVersion: String
+      KmsKeyId: String
+      MasterUsername: String
+      MasterUserPassword: !Ref ParentMasterPassword
+      Port: Integer
+      PreferredBackupWindow: String
+      PreferredMaintenanceWindow: String
+      SnapshotIdentifier: String
+      StorageEncrypted: Boolean
+      VpcSecurityGroupIds:
+        - String
+---
+Parameters:
+  ParentMasterUsername:
+    Description: 'username'
+    Type: String
+    Default: 'username!'
+Resources:
+  NewAmpApp-03:
```

Comment:
Resources names must be alphanumeric only.

This document is not a valid sample. Check the bottom of the page of the [AWS Cloudformation template reference documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-docdb-dbcluster.html
) for DocDB DBCluster samples.

Use [linters](https://github.com/martysweet/cfn-lint) and/or [YAML validators](https://jsonformatter.org/yaml-validator/9257f9) to validate the samples